src/benchmarklib/benchmark_utils.cpp

```diff
@@ -190,9 +191,15 @@ BenchmarkConfig CLIConfigParser::parse_basic_options_json_config(const nlohmann:
   }
   const Duration warmup_duration = std::chrono::duration_cast<opossum::Duration>(std::chrono::seconds{warmup});
 
-  return BenchmarkConfig{benchmark_mode,       verbose,  chunk_size,       *encoding_config, max_runs, timeout_duration,
-                         warmup_duration,      use_mvcc, output_file_path, enable_scheduler, cores,    clients,
-                         enable_visualization, out};
+  const auto validate = json_config.value("validate", default_config.validate);
+  if (validate)
+    out << "- Automatically validating results with SQLite. This will make the reported numbers invalid." << std::endl;
```

Comment:
The reported numbers a.k.a. the results? In that case maybe "... This will make the benchmark results invalid."?